includes/core.php

```diff
@@ -1127,7 +1127,7 @@ function get_default_copyright() {
 	 * @param string $copyright The default text for copyright.
 	 */
 	/* translators: the theme author */
-	return (string) apply_filters( 'go_default_copyright', get_bloginfo( 'name' ) );
+	return (string) apply_filters( 'go_default_copyright', sprintf( esc_html__( 'Go by %s', 'go' ), 'GoDaddy' ) );
```

Comment:
@ehamwey I don't think we need to pass this string through `esc_html__()` since its just a helper method for retrieving the copyright text. Generally it should be sanitized on output, which we're doing in certain places around Go.

We can probably just stick to `sprintf( __( 'Go by %s', 'go' ), 'GoDaddy' )`.

https://github.com/godaddy-wordpress/go/blob/ffd76753000c4b58bde5d274b6ad06c5c595620b/includes/customizer.php#L397

https://github.com/godaddy-wordpress/go/blob/ffd76753000c4b58bde5d274b6ad06c5c595620b/includes/template-tags.php#L566